ansible/roles/flannel/tasks/config.yml

```diff
@@ -13,7 +13,7 @@
   delegate_to: "{{ groups['etcd'][0] }}"
 
 - name: Load the flannel config file into etcd
-  shell: "/usr/bin/etcdctl --no-sync --peers={{ peers }} set {{ conf_loc }} < {{ conf_file }}"
+  shell: "etcdctl --no-sync --peers={{ peers }} set {{ conf_loc }} < {{ conf_file }}"
```

Comment:
hmm, it is in /usr/bin, since I used etcd playbook in contrib to deploy etcd. Not really remember the initial reason I removed the path here, will revert this line.